GVRf/Framework/framework/src/main/jni/objects/textures/render_texture.cpp

```diff
@@ -23,215 +23,438 @@
 #include "eglextension/msaa/msaa.h"
 
 namespace gvr {
-RenderTexture::RenderTexture(int width, int height, GLTexture* tex) :
+GLFrameBuffer *RenderTexture::renderTexture_gl_read_buffer = nullptr;
```

Comment:
Java_org_gearvrf_GVRViewManager_readRenderResultNative is a special case and modifying RenderTexture to support something in Java_org_gearvrf_GVRViewManager_readRenderResultNative in not desirable.

Attaching a proposal diff: [diff.txt](https://github.com/Samsung/GearVRf/files/1234802/diff.txt).

renderTexture_gl_read_buffer becomes instance member and should be properly deleted also. Java_org_gearvrf_GVRViewManager_readRenderResultNative deals with the problem independently and also takes care of the case where the process lingers but new activities (and egl contexts) get created. Based on the glIsFramebuffer docs it looks like glBindFramebuffer needs to be inserted in GLFrameBuffer also.